### PR TITLE
Change standard timezone to PST

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #860:*
https://github.com/bcgov/met-public/issues/860

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Comment by tiagograf-AOT** [comment]
This will impact anywhere the format function is used, from what I noticed this problem only happens with the scheduling date/time, can you check if there’s any impact in the rest of the app?

File: met-web/src/components/common/dateHelper.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const formatInTimeZone = (date: string, fmt: string, tz: string) =>
 
 export const formatDate = (d: string, formatString = 'yyyy-MM-dd') => {
     if (d) {
```
Comment (jadmsaadaot) [inline met-web/src/components/common/dateHelper.tsx:13]: If along the way we can change this variable name from d to date it would be great.

**Comment by jadmsaadaot** [comment]
Test failed since the format is outputting a date more accurate to us while test still expects "2022-09-17"

**Review by saravanpa-aot — CHANGES_REQUESTED** [review]
Is there a way to handle without specifying time zone?
Normally moment will have a way to convert to local timezone instead of specifying PST.. it shud ideally dependent on users time zone i guess

**Pull request closed without merging**